tests/cases/conformance/es6/modules/exportClassWithDenyName.ts

```diff
@@ -0,0 +1,2 @@
+// @target: ES5
+export class Object {}
```

Comment:
also add:
- class expression
- export default class
- ambient class (should not error)